### PR TITLE
Add a session filter which allows users to see only level sessions

### LINKED ISSUE #432: Add a session filter which allows users to see only level sessions
## Kind (Required)

- Other

 ## Overview (Required)

- Add a filter that ｀Beginners｀ and ｀Intermediates/Expert｀ chip tags and that title is `Level`.
- Please put `Level` filter after simultaneous interpretation filter.
- You can use [forBeginners](https://github.com/DroidKaigi/conference-app-2019/blob/master/model/src/commonMain/kotlin/Session.kt#L33)

 ## Links	

- https://github.com/DroidKaigi/conference-app-2019/pull/429

### PR DESCRIPTION
**Pull request by kirimin**
## Issue
- close #432 

## Overview (Required)
- Add a session filter which allows users to see only level sessions.
- Add tests.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by kirimin** [comment]
I think that test is randomly failed?

**Comment by takahirom** [comment]
Can you update your branch from merge master?
It is improved 🙇 
https://github.com/DroidKaigi/conference-app-2019/pull/411

**Comment by jmatsu** [comment]
Didn't your branch diverge from the current latest master? 
#411 may fix it, but please let us know if you still get this failure.

**Comment by kirimin** [comment]
thanks guys!!! 🙏

File: model/src/commonMain/kotlin/Filters.kt [hunk 1]
```diff
@@ -4,7 +4,8 @@ data class Filters(
     val rooms: Set<Room> = mutableSetOf(),
     val categories: Set<Category> = mutableSetOf(),
     val langs: Set<Lang> = mutableSetOf(),
-    val langSupports: Set<LangSupport> = mutableSetOf()
+    val langSupports: Set<LangSupport> = mutableSetOf(),
+    val audienceCategory: Set<AudienceCategory> = mutableSetOf()
```
Comment (jmatsu) [inline model/src/commonMain/kotlin/Filters.kt]: `audienceCategories` looks better.

File: model/src/commonMain/kotlin/SessionContents.kt [hunk 2]
```diff
@@ -6,10 +6,12 @@ data class SessionContents(
     val rooms: List<Room>,
     val langs: List<Lang>,
     val langSupports: List<LangSupport>,
-    val category: List<Category>
+    val category: List<Category>,
+    val audienceCategory: List<AudienceCategory>
```
Comment (jmatsu) [inline model/src/commonMain/kotlin/SessionContents.kt]: ditto

File: model/src/commonMain/kotlin/Filters.kt (hunk 1 above)
Comment (kirimin) [reply model/src/commonMain/kotlin/Filters.kt]: Thank you for reviewing!
I fixed them.

**Pull request merged**